book/src/checkpoint-sync.md

```diff
@@ -41,6 +41,13 @@ Once the checkpoint is loaded Lighthouse will sync forwards to the head of the c
 If a validator client is connected to the node then it will be able to start completing its duties
 as soon as forwards sync completes.
 
+### Use a community checkpoint sync endpoint
+
+The Ethereum community provides various [public endpoints](https://eth-clients.github.io/checkpoint-sync-endpoints/) for you to choose from for your initial checkpoint state. Select one for your network and use it as the url for the `--checkpoint-sync-url` flag.  e.g.
+```
+lighthouse bn --checkpoint-sync-url https://beaconstate.info/ ...
```

Comment:
I feel like People might just end up copying this line and all end up using this specific endpoint, which is sub-optimal from a safety perspective (no single checkpoint endpoint should be recommended). How about using a non-working domain instead, like `example.com`?

Reply:
that's a good point, I'm happy to merge that change if you or @remyroy would like to PR it (assuming no objections?)

Reply:
I'm good with @InvisibleSymbol suggestion. Let me do another PR.